### PR TITLE
[earcut-hpp] Update to 2.2.4

### PR DESCRIPTION
**Pull request by Osyotr**
Changes: https://github.com/mapbox/earcut.hpp/compare/v2.2.3...v2.2.4

### CODE REVIEW

File: ports/earcut-hpp/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,10 @@
+vcpkg_minimum_required(VERSION 2022-10-12) # for ${VERSION}
+
```
Comment (JonLiu1993) [inline ports/earcut-hpp/portfile.cmake:2]:
```suggestion
```

Comment (JonLiu1993) [reply ports/earcut-hpp/portfile.cmake:2]: @Osyotr, Thanks for your PR, we don't need to add this function now.

File: ports/earcut-hpp/portfile.cmake [hunk 2]
```diff
@@ -1,8 +1,10 @@
+vcpkg_minimum_required(VERSION 2022-10-12) # for ${VERSION}
+
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO mapbox/earcut.hpp
-    REF v2.2.3
-    SHA512 f87f0084dbbcda85b409c326587a376d443f2484b6a69c216d2ad94ea8ea238912dfe1174b464b08faec10ea5c29ebae6478e7abfe5ff682a7b043784c1e3acf
+    REF v${VERSION}
```
Comment (JonLiu1993) [inline ports/earcut-hpp/portfile.cmake:6]:
```suggestion
    REF "v${VERSION}"
```

**Comment by BillyONeal** [comment]
Thanks for the new version!

**Pull request merged**